net/openthread-br/Makefile

```diff
@@ -94,10 +96,8 @@ endef
 
 define Package/openthread-br/install
 	$(INSTALL_DIR) \
-		$(1)/etc/init.d \
 		$(1)/lib/netifd/proto \
-		$(1)/usr/sbin \
-		$(1)/var/lib/thread
```

Comment:
Why?

Reply:
But this one should be kept. It's where otbr-agent writes its data file.

Reply:
that's in conffiles. this change is in INSTALL_DIR. I don't think they're related.

Reply:
Having it in conffiles is to make sure it is kept during sysupgrade. Afaik it doesn't ensure the dir is there.

Reply:
OK added back after testing.

Reply:
Everything in `/var` can't be really kept as it's stored in the memory by default. Everything inside it will be lost after reboot.

https://github.com/openwrt/packages/pull/22252#issuecomment-1761024651
https://downloads.openwrt.org/snapshots/faillogs/aarch64_cortex-a53/base/base-files/compile.txt

Now we have a race condition issue because of it. In `base-files`, `/var` is soft linked to `/tmp`, but this Makefile creates a actual directory.

Reply:
Please don't necro old/closed issues/PRs. If this is currently a problem, open a new issue and mention the maintainer.

Reply:
In any case, this issue needs fixing.